### PR TITLE
Scott gen edits

### PR DESCRIPTION
**Pull request by borcuttjahns**
Let me know what you think about the shuffle. One thing to note is that with decreased point size, using dodge makes 6B slightly unreadable as it looks like a cloud of points. You may be able to mess with it more to get it to only dodge close points but right now they fan out regardless of what's close.

### CODE REVIEW

File: ckine/figures/figure3.py [hunk 1]
```diff
@@ -103,15 +99,15 @@ def PCA_receptor(ax, cell_name, datas):
 
     for kk in range(loadings.shape[1]):
         if kk == 3:
-            ax[1].scatter(0, 0, s=16, label='IL-15Ra')
```
Comment (aarmey) [inline ckine/figures/figure3.py]: Don’t like splitting up PCA plots. Maybe we can reduce A’s width?

File: ckine/figures/figure6.py [hunk 2]
```diff
@@ -70,14 +70,14 @@ def makeFigure():
     affComp(ax[0])
     calc_plot_specificity(ax[7], 'NK', df_spec, df_act, ckines, ckineConc_)
     calc_plot_specificity(ax[8], 'T-helper', df_spec, df_act, ckines, ckineConc_)
-    global_legend(ax[8], Spec=True, Mut=True)
```
Comment (aarmey) [inline ckine/figures/figure6.py]: How about get rid of the predicted EC50’s in B? We don’t talk about the model until later anyway. A parallel coordinates plot might be especially effective here, too: https://pandas.pydata.org/pandas-docs/version/0.9.0/visualization.html

**Comment by borcuttjahns** [comment]
Hey, made some changes to 3, and replaced 6B with a parallel coordinates plot. For 3, keeping the PCA plots together in combination with the legend is a bit tricky, and I'm starting to think the original format might be the best option we have.

**Comment by aarmey** [comment]
For Fig. 3, you could put the PCA plots down where the schematic is, then move it up to the top right corner? I agree that (A) should go back to double width.

**Comment by aarmey** [comment]
For Fig. 6A, I think we talked about making the x- and y-axes log-scaled.

**Comment by aarmey** [comment]
Last thing we should do on this PR—can you put wt IL-2 and IL-15 on the EC50 plot in Fig. 6?

**Comment by aarmey** [comment]
Two things I see (not for this PR)—the data in Fig. 6 really shows we need to fix the IL-2/IL-2Ra affinity. I'm also thinking we should "flip" Fig. 6. G–J can come first, then we talk about the muteins. Both these things we should handle on a separate PR.

**Comment by borcuttjahns** [comment]
Alright, I've proposed fixes for the issues you noted with this PR. I can get started on a new PR soon - we'll have to talk about how to fix the IL-2 IL2Ra affinity issue since fitting seemed to break with the new affinity, so I'd like to hear from you on how to attack that.

**Pull request merged**